platform/testvectors/json/test-config-missing.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
If I understand correctly. these files appear in the Review diffs, as they have moved location, not because there is some genuine edits, been done? Is that correct?

Reply:
Correct.